capycli/bom/create_components.py

```diff
@@ -259,6 +259,10 @@ def prepare_release_data(self, cx_comp: Component) -> Dict[str, Any]:
             # ensure that we have the only correct external-id name: package-url
             data["externalIds"]["package-url"] = cx_comp.purl.to_string()
 
+        # add information that this release was created by CaPyCli
+        data["additionalData"] = {}
+        data["additionalData"]["createdWith"] = "CaPyCli: " + capycli.get_app_version()
```

Comment:
I unfortunately have no time to test this now, but as this function is also called from update_release(), we have to be careful to not overwrite existing additionalData or cause unnecessary updates. But from reading the code it looks like this part of the data is ignored in update_release(), so it should be ok.